examples/vite-webpack-rspack/webpack/webpack.config.js

```diff
@@ -69,6 +69,14 @@ module.exports = {
       name: 'webpack',
       filename: 'remoteEntry.js',
       remotes: {},
+      dts: {
+        generateTypes: true,
```

Comment:
with this you can see it creates a dist folder with the types that should be synced to the folder in host